pkgs/development/libraries/uhttpmock/default.nix

```diff
@@ -11,10 +11,12 @@ stdenv.mkDerivation rec {
     sha256 = "0kkf670abkq5ikm3mqls475lydfsd9by1kv5im4k757xrl1br1d4";
   };
 
-  nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ autoconf gtk-doc automake libtool glib libsoup gobject-introspection ];
+  depsBuildBuild = [ gobject-introspection ];
```

Comment:
I don't think introspection belongs in depsBuildBuild, rather it belongs in nativeBuildInputs (with possibly pkg-config in depsBuildBuild).